app/helpers/application_helper.rb

```diff
@@ -34,7 +34,7 @@ def current_contractors_project?(project)
   def project_actions(status)
     case status
     when "Pending"
-      concat( link_to "Submit Payment", contractor_project_payment_path(@project), id: "checkout-button")
+      concat( link_to "Submit Payment", new_contractor_project_charge_path(@project), id: "checkout-button")
```

Comment:
Line is too long. [108/80]<br>Add parentheses to nested method call `link_to "Submit Payment", new_contractor_project_charge_path(@project), id: "checkout-button"`.<br>Space inside parentheses detected.